etc/docker/travis/alembic_mysql.ini

```diff
@@ -24,7 +24,7 @@ script_location = lib/rucio/db/sqla/migrate_repo/
 # the 'revision' command, regardless of autogenerate
 # revision_environment = false
 
-sqlalchemy.url = mysql://root:secret@mysql/mysql
+sqlalchemy.url = mysql+pymysql://root:secret@mysql/mysql
```

Comment:
MySQL-python seems to be not python3 compatible and was installed in the Dockerfiles. I removed the installation and changed the driver to pyMySQL which gets already installed by setup.py.